### DISCUSSION TITLE
How to uninstall it?My adapter still doesn't work after this driver installed, but I can't uninstall it though I have deleted the application and the kexts.

### DISCUSSION THREAD

**Discussion by MoePunchQWQ** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#274] [Q&A] [answered]
As the title, I don't know how to delete it. its really annoying that there's always an icon on my finder

**Comment by chris1111** [comment, marked as answer]
Seriously
![Capture d’écran, le 2022-12-19 à 14 55 16](https://user-images.githubusercontent.com/6248794/208509152-53ea26ef-d2a7-4fe5-abd9-c2d635a8528d.png)
